packages/api/src/storage.ts

```diff
@@ -226,30 +282,17 @@ export default class Storage {
   }
 
   /**
-   * @example <caption>Storage directory structure</caption>
```

Comment:
Removed this because I don't want to have to maintain it alongside the same tree in `docs/how-it-works.md`.